src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,43 @@
 import React from 'react';
-
+import { Movie } from '../../types/movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <>
```

Comment:
1 rule of the [checklist](https://github.com/mate-academy/react_movies-list/blob/master/checklist.md)